addon/components/bs-modal.js

```diff
@@ -278,7 +291,6 @@ export default class Modal extends Component {
    * @public
    */
   @arg
-  backdropTransitionDuration = 150;
```

Comment:
As per above, I assume you had removed this line by accident?

Reply:
Yes exact!!!